src/app/js/fields/wizard/index.js

```diff
@@ -121,27 +119,16 @@ const FieldEditionWizardComponent = ({
         {
             label: 'field_wizard_tab_identity',
             id: 'tab-identity',
-            component: <TabIdentity field={field} />,
-        },
-        {
-            label: 'field_wizard_tab_value',
-            id: 'tab-value',
             component: (
-                <TabValue
+                <TabGeneral
+                    field={field}
```

Comment:
Wich field ? From that naming, i can't understand what is the point of this prop.